### ISSUE TITLE
Add apkbuild-shellcheck?

### ISSUE THREAD

**Issue by ollieparanoid** [maxice8/atools#22]
atools is great and we began using apkbuild-lint outside of Alpine, in the pmaports repository of postmarketOS.

It would be great if we could also use apkbuild-shellcheck in the future, however the code for that is in the docker image " apkbuild-lint-tools" and not part of atools:

https://gitlab.alpinelinux.org/alpine/infra/docker/apkbuild-lint-tools/tree/master

Are there plans to move apkbuild-shellcheck to atools, and/or would you accept patches to do this?

**Comment by maxice8** [comment]
That is something @Ikke deals with it

We are also on GitLab now gitlab.alpinelinux.org/Leo/atools, can you open it there and tag him (should be @kdaudt) ?

**Comment by ollieparanoid** [comment]
sure

**Comment by ollieparanoid** [comment]
https://gitlab.alpinelinux.org/Leo/atools/issues/25

FWIW, if this repository should not be used anymore, it might be worth archiving it and leaving a link to the new gitlab repo.

**Comment by maxice8** [comment]
Yes, i just completely forgot about this repo once i moved to gitlab
